packages/solidity-contracts/scripts/hardhat/depositETH.ts

```diff
@@ -0,0 +1,51 @@
+import { Wallet, parseEther } from 'ethers';
+import type { Signer } from 'ethers';
+import { isB256, isBech32, toB256 } from 'fuels';
+import { task } from 'hardhat/config';
+
+task('depositETH', 'grants a given role to a given adress')
```

Comment:
wrong description: 

`task('depositETH', 'grants a given role to a given adress')`

Reply:
Solved